DRODLib/Serpent.cpp

```diff
@@ -68,7 +68,7 @@ const
 		return true;
 	}
 
-	if (!CanFindSwordsman())
+	if (!this->pCurrentGame->bBrainSensesSwordsman && !CanFindSwordsman())
```

Comment:
I don't think this change is necessary.  Isn't it redundant with the above if check?

Reply:
Good point, reverted.